src/lex.cpp

```diff
@@ -606,6 +606,10 @@ const char8* lexer::parse_string_literal() noexcept {
           ++c;
           break;
         }
+      case 'x':
+        // TODO (angel): do stuff here to identify invalid hex escape sequences?
+        // use parse_hexadecimal_number?
```

Comment:
looked at parse_hexadecimal_number more closely and this doesn't seem likely, just wanted to avoid writing redundant code